api/policies/passport.js

```diff
@@ -28,6 +28,7 @@ module.exports = function (req, res, next) {
     passport.session()(req, res, function () {
       // Make the user available throughout the frontend
       res.locals.user = req.user;
+      res.locals.VERSION = require('../../package').version;
```

Comment:
Can you do the import at the top, so we don't keep bringing the whole object into memory.

So "var version = require('../../package').version" and then just use that variable here.